Update pause-on-lock

adrianvg opened the pull request:
Added support for Mate DE.
Tested with Ubuntu 18.04.2 LTS and Mate DE 1.20.1.

adrianvg committed:
Update pause-on-lock

Added support for Mate DE.

adrianvg reviewed (commented):
@folixg Did the PR you asked for. Please have a look at it.
Thanks in advance and also for providing the original script!

adrianvg commented:
Wait a sec, the diff when merged with your code, doesn't look proper.
It jumbled the lines for some reason and put the Mate code in the middle of the X-cinnamon.
Check my fork for clarity. Thanks.

adrianvg committed:
Update pause-on-lock

Mix-up when changed line 37 from X-Cinnamon to mate.

adrianvg committed:
Update pause-on-lock

And a typo...

folixg commented:
Looks good so far. It would be perfect, if you also update the README.

adrianvg commented:
Will look into the readme during the day.
Thanks!

adrianvg committed:
Update README.md

Added text for Mate DE support.

adrianvg commented:
Added text and link for Mate in the readme.

adrianvg reviewed (commented):
@folixg Have a look please. Thanks.

folixg commented on pause-on-lock:
```diff
@@ -33,6 +33,10 @@ elif [ "$XDG_CURRENT_DESKTOP" == "X-Cinnamon" ] ; then
+  LOCK_SIGNAL="true"
+  UNLOCK_SIGNAL="false"
 else
 echo "Error: Unsupported desktop. Supported desktops are Unity, GNOME and Cinnamon."
```
One last line to change, after that everything should be fine.
```suggestion
echo "Error: Unsupported desktop. Supported desktops are Unity, GNOME, Cinnamon and MATE."
```

adrianvg replied:
Done!

adrianvg committed:
Update pause-on-lock

Added Mate to:
echo "Error: Unsupported desktop. Supported desktops are Unity, GNOME, Cinnamon and Mate."

folixg commented:
Thanks @adrianvg!

adrianvg commented:
> Thanks @adrianvg!

No prob, glad to help. Self-interest you know. ;-)
